sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs

```diff
@@ -63,10 +63,9 @@ public class ServiceBusReceiver : IAsyncDisposable
         public virtual int PrefetchCount { get; }
 
         /// <summary>
-        /// Gets the ID to identify this client. This can be used to correlate logs and exceptions.
+        /// A unique name used to identify the processor.  If <c>null</c> or empty, a random unique value will be will be used.
```

Comment:
nit:  We should consider updating this block to remove the implied guarantee about uniqueness, since we don't have control over the value in all cases.